### ISSUE TITLE
Run bbc-a11y on PGLs during build

### ISSUE THREAD

**Issue by jamesdonoh** [bbc/simorgh#5411]
**Is your feature request related to a problem? Please describe.**
Currently we aren’t running `bbc-a11y` on CI/CD for any PGL assets. This increases the chance we could accidentally make photo galleries not accessible.

**Describe the solution you'd like**
- Ensure that this tool is run on one PGL and that any failures cause the build to fail.
- The bbc-a11y module uses `./bbcA11y.js` to configure its run. Add the PGL page type here.

Note:
- Investigate whether this test should also be run against `test` - if so, another issue should be raised to amend this file `simorgh-infrastructure/config/articles.txt`.

**Describe alternatives you've considered**
None.

**Testing notes**
- Run `npm run dev && npm run bbcA11y` to determine a11y tests are being run on PGL assets.
- Also verify it is being run correctly on CI and will cause the build to fail.

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**
- https://github.com/bbc/bbc-a11y
